packages/components/src/DescriptionList/DescriptionList.tsx

```diff
@@ -4,17 +4,18 @@ import { Typography } from "../Typography";
 
 interface DescriptionListProps {
   /**
-   * A tuple where the first item is the string to display as the term
-   * and the second value is the string to display as the definition
+   * A tuple where the first item is the string to display as the term,
+   * the second value is the string to display as the definition,
+   * and the third value is an optional key to use as the React key in scenarios where the display value is not unique for all elements in the data set.
    */
-  data: [string, string | ReactNode][];
+  readonly data: [string, string | ReactNode, string?][];
 }
 
 export function DescriptionList({ data }: DescriptionListProps) {
   return (
     <dl className={styles.descriptionList}>
-      {data.map(([term, description]) => (
-        <div key={term} className={styles.termGroup}>
+      {data.map(([term, description, key]) => (
+        <div key={key || term} className={styles.termGroup}>
```

Comment:
Hi, apologize for the delay, we were heads down on the retheme.

thoughts on using term + index instead of adding another value on the tuple? I understand the side effects of it but this is also a super simple list that it's okay to do.

Reply:
Thanks for the suggestion, @darryltec! I'll make that change.